app/Http/Controllers/SignLocallyController.php

```diff
@@ -235,8 +241,25 @@ public function downloadSignedFile(Request $request)
         }
 
         $data = $this->eidEasyApi->downloadSignedFile($docId);
+        $fileName = $data['filename'];
+
+        $signedFilesContent = [];
+        foreach ($data['signed_files'] as $file) {
+            $signedFilesContent[] = base64_decode($file['contents']);
+        }
+
+        if (count($signedFilesContent) > 1) {
+            $zipDto = $this->zipService->zipPdfs($fileName, $signedFilesContent);
```

Comment:
how do we know that files are PDFs?

Reply:
Thats a good question, but there isnt good way to tell.
Currently our (eid-sample-app) application doesnt have option to have many files in array other than PDFs